language/fr/converter.lang

```diff
@@ -1,5 +1,96 @@
 <?php
 
-include("{$_EnginePath}language/en/converter.lang");
+$_Lang['sys_attack_title']      = 'Les flottes suivantes se sont affrontées le %s';
+
+$_Lang['Conv_att_user']         = 'Attaquant [color=red][b]%s[/b][/color] %s [%s:%s:%s]';
+$_Lang['Conv_tech']             = 'Armes : xXx% | Bouclier : xXx% | Coque : xXx%';
+$_Lang['Conv_fleet']            = '[b]Flottes :[/b]';
+$_Lang['Conv_defense']          = '[b]Défenses :[/b]';
+$_Lang['Conv_def_user']         = 'Défenseur [color=green][b]%s[/b][/color] %s [%s:%s:%s]';
+$_Lang['Conv_def_abandoned_1']  = 'de la planète abandonnée';
+$_Lang['Conv_def_abandoned_3']  = 'de la lune abandonnée';
+
+$_Lang['Conv_in_next_round']    = '[i][b]In the next round[/b]...[/i]';
```

Comment:
Looks like these two strings have not been translated.

Reply:
You are right. Fixed.